api/types/status.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 type Status struct {
-	State       State     `json:"state"`
+	State       State     `json:"state" enums:"Pending,Running,Succeeded,Failed" example:"Succeeded"`
```

Comment:
The only change required for we to run the whole flow but if we happen to improve our open api spec through examples data would be more accurate ofc.